tests/end2end/data/navigate/multilinelinks.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Navigate multiline links</title>
+    </head>
+    <body>
+        <p>Test page</p>
+        <a href="/data/navigate/prev.html">previous</a>
+        <a href="/data/numbers/5.txt">mid</a>
+        <a href="/data/navigate/next.html">next</a>

```

Comment:
There isn't actually a multi-line link in there - you can remove the `prev.html` and `next.html` links here, and then name `mid` something like `next<br/>page` (`<br/>` is a [line break in HTML](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br))